cfme/v2v/migrations.py

```diff
@@ -125,14 +126,14 @@ def fill(self, values):
                        ...
                     }
         """
-        for mapping in values['mappings']:
-            self.source_clusters.fill(mapping['sources'])
-            self.target_clusters.fill(mapping['target'])
-            self.add_mapping.click()
+        if values:
+            for mapping in values['mappings']:
+                self.source_clusters.fill(mapping['sources'])
+                self.target_clusters.fill(mapping['target'])
+                self.add_mapping.click()
+
         was_change = not self.mappings_tree.is_empty
-        if was_change:
```

Comment:
**Question**: Why should we remove this check?

Reply:
@kedar : edit form does not work with 'was_change' . only some values are changed while editing and hence the form does not change , but we still need to click on next button

Reply:
Did you try to run any other tests with these changes? We don't want this to break other things. Maybe try to run other UI tests and see if they still work.